src/pkgdev/const.py

```diff
@@ -40,3 +40,8 @@ def _GET_CONST(attr, default_value):
 USER_CONF_FILE = os.path.join(getattr(_module, "USER_CONFIG_PATH"), "pkgdev.conf")
 SYSTEM_CONF_FILE = "/etc/pkgdev/pkgdev.conf"
 BUNDLED_CONF_FILE = os.path.join(DATA_PATH, "pkgdev.conf")
+
+BZ_FIXED = "fixed"
```

Comment:
please define it using python's `enum.Enum`, you can get all the know values from it